lib/tessel/wifi.js

```diff
@@ -60,9 +64,10 @@ Tessel.prototype.getWifiInfo = function() {
 };
 
 function compareBySignal(a, b) {
-  if ((a.quality / a.quality_max) > (b.quality / b.quality_max)) {
+  // eval(quality) turns a string fraction like 21/70 into a decimal
+  if (eval(a.quality) > eval(b.quality)) {
     return -1;
-  } else if ((a.quality / a.quality_max) < (b.quality / b.quality_max)) {
+  } else if (eval(a.quality) < eval(b.quality)) {
```

Comment:
I saw the `evil` addition to the `.jshintrc` but wasn't sure why. Missed that as well.